typescript-expressions/utils/1.0.0/tests/utils__1.0.0.test.ts

```diff
@@ -87,4 +88,26 @@ describe('Test is_not_null utils', function() {
     })
 })
 
+// boolean_to_input Test
```

Comment:
boolen_to_int Test?

Reply:
Yep, can you elaborate on your question?

Reply:
I was checking if it is suppose to be `boolean_to_input` or `boolean_to_int` in this comment?

Reply:
Fixed in e4acc1abf573d2ccf6d8379b633cf8e16b3a82ae